CHANGELOG.rst

```diff
@@ -2,44 +2,81 @@
-  this is now ``#b0b0b0``.
-* ``--utrecht-action-disabled-cursor``. This does not take the value of an old token. It controls
-  the looks of the cursor when hovering a disabled button. For the Open Forms theme, this is now
-  ``not-allowed``.
-* ``--utrecht-action-submit-cursor``. This does not take the value of an old token. It controls the
-  looks of the cursor when hovering over a submit button. For the Open Forms theme, this is now
-  ``pointer``.
+* ``--utrecht-button-primary-action-danger-focus-border-color`` has ``#000000`` (black)
+  in the Open Forms theme.
+* ``--utrecht-button-secondary-action-danger-background-color`` takes the value of the
+  old ``--of-button-danger-bg``.
+* ``--utrecht-button-secondary-action-danger-color`` takes the value of the old
+  ``--of-button-danger-fg``.
+* ``--utrecht-button-secondary-action-focus-border-color`` takes the value of the old
+  ``--of-color-focus-border``.
+* ``--utrecht-button-subtle-danger-color``  takes the value of ``--of-color-danger``.
+* ``--utrecht-button-subtle-danger-background-color``  takes the value of
+  ``--of-color-bg``.
+* ``--utrecht-button-subtle-danger-hover-background-color`` takes the value
+  ``--of-color-bg``.
+* ``--utrecht-button-subtle-danger-active-background-color`` takes the value of the old
+  ``--of-button-danger-active-bg``.
+* ``--utrecht-button-disabled-color``. This does not take the value of an old token. For
+  the Open Forms theme this is now ``#ffffff``.
+* ``--utrecht-button-disabled-background-color``. This does not take the value of an old
+  token, the colour was previously obtained by graying out the primary button. For the
+  Open Forms theme, this is now ``#b0b0b0``.
+* ``--utrecht-action-disabled-cursor``. This does not take the value of an old token. It
+  controls the looks of the cursor when hovering a disabled button. For the Open Forms
+  theme, this is now ``not-allowed``.
+* ``--utrecht-action-submit-cursor``. This does not take the value of an old token. It
+  controls the looks of the cursor when hovering over a submit button. For the Open
+  Forms theme, this is now ``pointer``.
+
+Additionally, in the ``.openforms-theme`` we apply some custom CSS overrides that may
+now need to be replicated in your own theme since they're now scoped to our own theme
+selector.
+
+Unfortunately, setting up a backwards compatible layer was considered too complex.
+
+**Buttons that look like links**
+
+These are now actual links instead of button elements. If you have automated test
+scripts, they may fail on these links now when querying by accessible role.
+
+**Formio time component cleanup [#3531]**
+
+The time component min/max time validation is moved into the ``validate`` namespace, for
+a consistent builder configuration.
+
+Existing component definitions need to be updated: ``component.minTime`` becomes
+``component.validate.minTime``, and a similar action is needed for ``maxTime``.
```

Comment:
Isn't this done with a migration? From this it seems like the form designers need to make the change?